spec/support/log.cr

```diff
@@ -1,33 +1,9 @@
 require "log"
+require "log/spec"
 
-def capture_logs(source, level : Log::Severity = :debug)
-  log = Log.for(source)
-  old_backend = log.backend
-  old_level = log.level
-  log.backend = mem_backend = Log::MemoryBackend.new
-  log.level = level
-  begin
-    yield mem_backend.entries
-    mem_backend.entries
-  ensure
-    log.backend = old_backend
-    log.level = old_level
-  end
+# Turn off logging in specs
+def log_setup_on_specs
+  Log.setup(:none)
```

Comment:
Should this file be removed completely? Maybe when `"log/spec"` is required it might disable all the logs by default.

Reply:
There is one more call of `log_setup_on_specs` in the `Log.setup` specs that is needed to restore the "tuning off" of the logs.

I wasn't sure if doing a `Log.setup(:none)` in `"log/spec"` was right. If some module emit logs and `"log/spec"` is not required, the logs will be shown in stdout.

If we want to turn off logging in specs by default I would suggest doing that in `"spec"` module directly. Checking if `Log` is defined to avoid requiring `"log"` in the first place.

Reply:
I see, maybe `spec` should set some flag or something so logs can be disabled by default when testing?